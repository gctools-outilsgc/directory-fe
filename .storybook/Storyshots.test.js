import React from 'react';
import ReactDOM from 'react-dom';

import initStoryshots, { multiSnapshotWithOptions }
  from '@storybook/addon-storyshots';

ReactDOM.createPortal = node => React.createElement("portal-dummy", null, node);

initStoryshots({
    suite: 'FileProperties',
    test: multiSnapshotWithOptions({}),
});
