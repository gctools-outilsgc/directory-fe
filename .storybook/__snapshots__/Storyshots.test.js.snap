// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Loading With default options 1`] = `
<div
  className="card"
  style={
    Object {
      "border": "none",
      "boxShadow": "none",
      "width": "100%",
    }
  }
>
  <div
    className="card-body"
  >
    <div
      className="card-title"
    >
      <div
        className="loading-state fake-head mb-3"
      />
    </div>
    <span
      className="sr-only"
    >
      Loading ...
    </span>
    <div
      className="row"
    >
      <div
        className="col"
      >
        <div
          className="loading-state mt-3"
          style={
            Object {
              "height": "18px",
              "marginBottom": "3px",
              "width": "65px",
            }
          }
        />
        <div
          className="loading-state"
          style={
            Object {
              "height": "16px",
              "width": "150px",
            }
          }
        />
      </div>
      <div
        className="col"
      >
        <div
          className="loading-state mt-3"
          style={
            Object {
              "height": "18px",
              "marginBottom": "3px",
              "width": "65px",
            }
          }
        />
        <div
          className="loading-state"
          style={
            Object {
              "height": "16px",
              "width": "150px",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots UserAvatar Authenticated User with avatar and editing 1`] = `
<div>
  <div>
    <div
      className="query"
    >
      <div>
        <img
          alt="No photo"
          src="https://avatars0.githubusercontent.com/u/7603237?s=460&v=4"
        />
      </div>
      <div
        className="mutate"
      >
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
        >
          <label
            htmlFor="avatarUploadTest"
          >
            Upload
            <input
              id="avatarUploadTest"
              onChange={[Function]}
              required={true}
              style={
                Object {
                  "display": "none",
                }
              }
              type="file"
            />
          </label>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots UserAvatar Authenticated User with no avatar and editing 1`] = `
<div>
  <div>
    <div
      className="query"
    >
      <div>
        No AVATAR
      </div>
      <div
        className="mutate"
      >
        <button
          aria-label={null}
          className="btn btn-secondary"
          onClick={[Function]}
        >
          <label
            htmlFor="avatarUploadTest"
          >
            Upload
            <input
              id="avatarUploadTest"
              onChange={[Function]}
              required={true}
              style={
                Object {
                  "display": "none",
                }
              }
              type="file"
            />
          </label>
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots UserAvatar Authenticated User with no avatar and no editing 1`] = `
<div>
  <div>
    <div>
      No AVATAR
    </div>
  </div>
</div>
`;

exports[`Storyshots UserAvatar Not logged in not own profile 1`] = `
<div>
  <div>
    <div>
      No AVATAR
    </div>
  </div>
</div>
`;

exports[`Storyshots UserAvatar With default options 1`] = `
<div>
  <div>
    <div>
      No AVATAR
    </div>
  </div>
</div>
`;
